src/modules/medical-equipment/request/calibration/calibration-request-create.tsx

```
@@ -1,6 +1,189 @@
+    }
+  };
+
+  const onsubmit = async (data: z.infer<typeof formSchema>) => {
+    setLoading(true);
+    setError(null);
+
+    try {
+      const submissionDate = new Date().toISOString();
+      await createCalibrationRequest({...data, submissionDate});
+      router.push(`/dashboard/medical-equipment/${equipmentId}?success=true`);
+    } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)